src/main/java/com/choelinska/horrorlibz/services/impl/UserServiceImpl.java

```diff
@@ -25,12 +25,24 @@ public UserServiceImpl(UserRepository userRepository, ModelMapper modelMapper){
 
     @Override
     public void saveUser(UserTo user) {
-        userRepository.save(mapToEntity(user));
+        userRepository.save(this.mapToEntity(user));
+    }
+
+    @Override
+    public List<UserEntity> getSortedByName(){
+        return this.userRepository.findAll(sortByNameDesc());
```

Comment:
Since the controller must return a TO, the service needs to provide it instead of the entity. Change the return type of this method and map the entity to TO inside it.